pkg/storage/tpr/storage_interface_test.go

```diff
@@ -141,6 +141,9 @@ func TestCreateWithNoNamespace(t *testing.T) {
 		Spec: sc.BrokerSpec{
 			URL: "http://my-awesome-broker.io",
 		},
+		Status: sc.BrokerStatus{
+			Conditions: []sc.BrokerCondition{},
```

Comment:
It seems there was some change which affected treating `nil` arrays in JSON encoding/decoding.
Without this change (and other changes in this fail), `deepCompare` fails with error:
```
input and output are different: 
  object.Status.Conditions:
    a: []servicecatalog.BrokerCondition(nil)
    b: []servicecatalog.BrokerCondition{}
```